client/me/purchases/vat-info/style.scss

```
@@ -27,3 +27,59 @@
 .vat-info__country-select {
 	width: 100%;
 }
+
+.vat-form__field-wrapper {
+	display: flex;
+	border-radius: 3px;
+
+	&:focus-within {
+		box-shadow: 0 0 0 2px var(--color-primary-20);
+		outline: 1px solid var(--color-primary);
+
+		& > .vat-field__overlay-prefix {
+			border-top-color: transparent;
+			border-left-color: transparent;
+			border-bottom-color: transparent;
+		}
+
+		& > input[type="text"].form-text-input {
+			border: none;
+		}
+	}
+
+	&--isError {
```

This should probably use snake-case like the other names.

```suggestion
	&--error {
```